src/test/mochitest/browser_dbg-editor-gutter.js

```diff
@@ -46,4 +46,13 @@ add_task(async function() {
   await waitForDispatch(dbg, "REMOVE_BREAKPOINT");
   is(getBreakpoints(getState()).size, 0, "No breakpoints exist");
   assertEditorBreakpoint(dbg, 4, false);
+
+  // Ensure that clicking the gutter removes all breakpoints on a given line
+  await addBreakpoint(dbg, source, 4, 0);
+  await addBreakpoint(dbg, source, 4, 1);
+  await addBreakpoint(dbg, source, 4, 2);
+  clickGutter(dbg, 4);
+  await waitForDispatch(dbg, "REMOVE_BREAKPOINT", 3);
+  is(getBreakpoints(getState()).size, 0, "No breakpoints exist");
+  assertEditorBreakpoint(dbg, 4, false);
```

Comment:
Nice!

Reply:
It'd be nice to also have a jest unit test for `toggleBreakpointsAtLine`, which will help us refactor that action if we ever touch it in the future